constants.py

```diff
@@ -238,6 +238,18 @@ def with_variables(self, variables: JsonType) -> GQLOperation:
 
 
 GQL_OPERATIONS: dict[str, GQLOperation] = {
+    # retuns PlaybackAccessToken_Template, for fix 2024/5
+    "PlaybackAccessToken": GQLOperation(
+        "PlaybackAccessToken",
+        "3093517e37e4f4cb48906155bcd894150aef92617939236d2508f3375ab732ce",
```

Comment:
Where is this token from and why is it hardcoded?

Reply:
This is a SHA256 digest of the GQL operation, not any particular token. If you look into the constants file, there's like 10 other operations defined, all with their own SHA256 digest included.